docs/advanced/retry-operations.md

```diff
@@ -79,9 +79,7 @@ const client = createClient({
     cacheExchange,
     retryExchange({
       retryIf: error => {
-        if ((error && error.graphQLErrors.length > 0) || error.networkError) {
-          return true;
-        }
+        return Boolean(error && (error.graphQLErrors.length > 0 || error.networkError));
```

Comment:
The `retryIf` function is guaranteed to have an error passed to it https://github.com/FormidableLabs/urql/blob/ae650d103f7de7c266d8b0c5f9a489991bede1e4/exchanges/retry/src/retryExchange.ts#L110,

the error here is probably a mistake in the braces, i.e.

```suggestion
        return !!error && (error.graphQLErrors.length > 0 || error.networkError));
```

Reply:
Oh right, in that case I think we can just do

return error.graphQLErrors.length > 0 || error.networkError;